src/Guzzle6HttpAdapter.php

```diff
@@ -45,24 +45,40 @@ public function sendRequest(RequestInterface $request)
     {
         try {
             return $this->client->send($request);
-        } catch (RequestException $e) {
-            throw $this->createException($e);
+        } catch (Guzzle\SeekException $e) {
+            throw new RequestException($e->getMessage(), $request, $e);
+        } catch (Guzzle\TransferException $e) {
```

Comment:
Guzzle got 2 types of Exceptions. SeekException and TransferException. Both are implementing GuzzleException. Logically the same but sure, I'll make the change.